src/main/java/com/kamilrafalko/parkingsystem/domain/PaymentDue.java

```diff
@@ -0,0 +1,27 @@
+package com.kamilrafalko.parkingsystem.domain;
+
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
```

Comment:
Jak wysyłasz coś do review to wszelkie importy z wildcardami są niemile widziane